units/src/fee_rate/mod.rs

```diff
@@ -10,7 +10,7 @@ use core::{fmt, ops};
 #[cfg(feature = "arbitrary")]
 use arbitrary::{Arbitrary, Unstructured};
 
-/// Represents fee rate.
+/// A fee rate.
```

Comment:
In fd3585f1c06e9c82ad6dbf59f557a52e854d9798:

This should be just `Fee rate.` to be consistent with the rest.

Reply:
Done